### PR TITLE
Add SETA awarness to mozci

### LINKED ISSUE #366: Give SETA awareness to mozci
Whenever we fetch a new allthethings.json we should also determine the schedule at which SETA runs.
This is useful for automatic backfilling since we want to know how far we need to go to find an issue.

For instance, our current SETA skipping level is of 7 pushes instead of 5 pushes [1]
The information is contained within the scheduler's name:
    "tests-fx-team-snowleopard-opt-unittest-7-3600":

For now, the same value is used across trees.

[1] https://github.com/mozilla/pulse_actions/blob/master/pulse_actions/handlers/backfilling.py#L25

### PR DESCRIPTION
**Pull request by F3real**
Fixes #366

### CODE REVIEW

File: test/test_platforms.py [hunk 1]
```diff
@@ -119,6 +121,34 @@ def test_include_exclude(self):
...
+    def test_parse_correctly(self, fetch_allthethings_data):
+        """get_SETA_interval_dict should return a dict with correct SETA intervals."""
+        fetch_allthethings_data.return_value = MOCK_ALLTHETHINGS
+        self.assertEquals(
+            get_SETA_interval_dict(),
+            {"Rev4 MacOSX Snow Leopard 10.6 fx-team debug test cppunit": [7, 3600]}
+        )
+
+
+class TestGetSETAInfo(unittest.TestCase):
```
Comment (armenzg) [inline test/test_platforms.py]: The only change I would do is to only have 1 class but two tests. You can call the class TestSETA.

**Comment by armenzg** [comment]
Hi @F3real this is great!
I would only do that one change I mentioned.
My apologies for having taken so long to review this. I hope that having dealt with other review faster could somewhat redeem me! sorry again.

**Comment by armenzg** [comment]
Let me know when you fix that class and I will merge this.

**Comment by F3real** [comment]
I've made changes. @armenzg

**Comment by armenzg** [comment]
Wonderful! Thanks!

**Pull request merged**